apps/site/assets/ts/stop/components/ExternalMapLink.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from "react";
+import renderFa from "../../helpers/render-fa";
+
+const ExternalMapLink = ({
+  address
+}: {
+  address: string;
+}): ReactElement<HTMLElement> => {
+  return (
+    <a
+      href={`https://maps.google.com/?q=${encodeURIComponent(address)}`}
```

Comment:
I know you're waiting on some feedback from Anna, so I'd love for you to refine this a bit more! A few ideas:

1. I remember the team also discussed refining the displayed result to be more indicative of the station we're looking at, rather than plainly showing an address. Example for North Station below - is there anything we can add to the URL to help us here?

<img width="1060" alt="image" src="https://user-images.githubusercontent.com/2136286/218579429-65508298-5e95-4f84-85b3-75f1463a1145.png">

2. Have you done any research on approaches to having links open in native mobile apps? This [page for opening links in Google Maps](https://developers.google.com/maps/documentation/urls/get-started) has a slightly different approach than that used here. Some other resources suggest that [an alternate script can be used for iOS devices](https://medium.com/@colinlord/opening-native-map-apps-from-the-mobile-browser-afd66fbbb8a4) - give those a try or follow up with some more research!

Reply:
1.  TLDR: It's possible but its going to be a lift because google and apple do things differently, and it might be best broken out into a separate ticket to deal with how google does things.  It looks like both services allow us to query by lat-long to get user friendly names.  Google allows searching by address, or station name as well.  The only catch is each service requires a different query param for friendly location names.  [Apple](https://developer.apple.com/library/archive/featuredarticles/iPhoneURLScheme_Reference/MapLinks/MapLinks.html) requires the query to be the station name, which is simple.  Google requires a `place_id` in order to show user friendly names as outlined [here](https://developers.google.com/maps/documentation/urls/get-started#search-examples).  This means we need to figure out what the `place_id` is for every station on [this page](https://developers.google.com/maps/documentation/places/web-service/place-id).  
https://www.google.com/maps/search/?api=1&query=42.413827136668104,-70.99165717317734&query_place_id=ChIJSbu0PwZu44kRLxNeyhVE0oI
https://www.google.com/maps/search/?api=1&query=Wonderland&query_place_id=ChIJSbu0PwZu44kRLxNeyhVE0oI
https://maps.apple.com/?q=Wonderland&ll=42.413827136668104,-70.99165717317734

2.  Thanks for the links! They helped a ton.

Reply:
Great investigation. Agree we should definitely leave some of this for a future ticket then. Storing the `place_id` for each is a really useful idea that we should figure out at some point.

> We can try pushing a version of just google maps urls outlined in the links above and someone with an iphone can test it, and we could go from there.

Yep, let's go ahead with that.